third_party/compilerworks/vertica_to_bq_migration/README.md

```diff
@@ -3,8 +3,21 @@
 This directory contains [user-defined functions](https://cloud.google.com/bigquery/docs/reference/standard-sql/user-defined-functions)
 which mimic the behavior of proprietary functions in Vertica. Each UDF within this
 directory will be automatically synchronized to the `bqutil` project within the
-`or` dataset for reference in queries.
+`vertica_to_bq` dataset for reference in queries.
 
 ## UDFs
 
-Nothing here yet, check back soon...
\ No newline at end of file
+* [next_day](#next_daydate_value-date-day_name-string)
+
+
+## Documentation
+
+
+### [next_day(date_value DATE, day_name STRING)](next_day.sql)
+This function calculates the next occurance of a specific day based off an initial date value. The function is based off of the Vertica (function)[https://www.vertica.com/docs/9.2.x/HTML/Content/Authoring/SQLReferenceManual/Functions/Date-Time/NEXT_DAY.htm] 
+```sql
+   SELECT bqutil.vertica_to_bq.next_day(Date('2016-04-29'),'mon'); 
+   SELECT bqutil.vertica_to_bq.next_day(Date('2016-04-29'),'fri');
```

Comment:
I don't believe we'll be hosting these UDFs in our bqutil project. @ryanmcdowell thoughts?